components/ReadMore.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import styled from "styled-components/native";
+import placeholder from "../assets/placeholder1.png";
+import { Share } from "react-native";
+import { StarRating } from "./StarRating";
+
+import homeBackground from "../assets/background.jpg";
+
+const ResultContainer = styled.ImageBackground`
+  flex: 1;
+  padding: 50px 15px 30px 15px;
+  align-items: center;
+  justify-content: center;
+`;
+
+const GroupImageAndText = styled.View`
+  flex-direction: row;
+  padding: 0 5px;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+const ImageContainer = styled.Image`
+  height: 125px;
+  width: 25%;
+  margin-right: 10px;
+  margin-bottom: 0;
+`;
+
+const GroupAuthorAndTitle = styled.View`
+  width: 75%;
+  flex-direction: column;
+`;
+
+const InfoText = styled.Text`
+  background: rgba(0, 0, 0, 0.5);
+  font-size: 14px;
+  color: #fff;
+  padding-bottom: 2px;
+`;
+
+const TitleText = styled(InfoText)`
+  font-size: 15px;
+  font-weight: 700;
+`;
+
+const ScrollViewContainer = styled.ScrollView`
+  padding: 20px 15px 0 15px;
+  height: 150px;
+  max-width: 400px;
+`;
+
+const DescriptionText = styled(InfoText)`
+  background: rgba(0, 0, 0, 0.5);
+  text-align: justify;
+`;
+
+const GroupButtons = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 20px;
+`;
+
+const StyledButton = styled.TouchableOpacity`
+  height: 40px;
+  width: 46%;
+  max-width: 250px;
+  padding: 5px;
+  margin: 0 5px;
+  background: #43464b;
+  border-radius: 15px;
+  border: 1px solid white;
+  align-items: center;
+  justify-content: center;
+`;
+
+const ButtonText = styled.Text`
+  color: #fff;
+  text-transform: uppercase;
+  text-align: center;
+`;
+
+export const ReadMore = ({ navigation, route }) => {
+  const book = route.params.data;
+
+  const navigateToHomeScreen = () => {
+    navigation.navigate("Home");
+  };
+
+  const onShare = async () => {
+    try {
+      const result = await Share.share({
```

Comment:
great job with `async await` logic :)
For further explanation, you can check out https://medium.com/javascript-in-plain-english/async-await-javascript-5038668ec6eb